web-client/src/primitives/signature.rs

```diff
@@ -25,7 +25,34 @@ impl Signature {
     /// Serializes the signature to a byte array.
     #[wasm_bindgen(js_name = toBytes)]
     pub fn serialize(&self) -> Vec<u8> {
-        self.inner.as_bytes().to_vec()
+        self.inner.to_bytes().to_vec()
+    }
+
+    #[wasm_bindgen(js_name = fromAsn1)]
+    pub fn from_asn1(bytes: &[u8]) -> Result<Signature, JsError> {
```

Comment:
What is `asn1`?

Reply:
https://en.wikipedia.org/wiki/ASN.1

Reply:
The serialization format in which signatures are returned by the Webauthn API.